controllers/tab/issueEntry/form/IssueEntryPublicationMetadataForm.inc.php

```diff
@@ -300,7 +303,6 @@ function execute($request) {
 					$publishedArticle = $publishedArticleDao->newDataObject();
 					$publishedArticle->setId($submission->getId());
 					$publishedArticle->setIssueId($issueId);
-					$publishedArticle->setDatePublished(Core::getCurrentDate());
```

Comment:
The main reason why I remove the setDatePublished function here was because 'datePublished' doesn't belong to the submissions table any more (it belongs now to submission_settings).
So I'm not quite sure where to initialize the date_published for unpublished submissions...